genetic_algorithm/GeneticAlgorithm.cpp

```diff
@@ -13,16 +13,18 @@ GeneticAlgorithm::GeneticAlgorithm(
 		int geneSize,
 		double mutationRate,
 		double elitismRate
-	)
+)
 {
 	this->mutationRate = mutationRate;
 	this->elitismRate = elitismRate;
 	this->elitismCount = populationSize * elitismRate;
 
 	this->population = std::vector <Chromossome>(populationSize);
 	for (unsigned i = 0; i < this->population.size(); i++)
+	{
 		this->population[i] = Chromossome(geneSize);
-	this->bestChromossome = this->population[0]; // Any element will do
+		this->bestChromossome = this->population[0]; // Any element will do
+	}
```

Comment:
The `this->bestChromossome = this->population[0];` was in the correct spot! Move it one line bellow (outside the bracket) to fix.